BBHX_Phenom.py

```diff
@@ -54,29 +68,47 @@ def imr_duration(**params):
         time_length = params['t_obs_start']
     return time_length * 1.1
 
-def interpolated_tf(m1, m2):
+
+def interpolated_tf(m1, m2, m_mode=None):
+    """Interpolate the time frequency-track.
+
+    Defaults to the dominant (2,2) mode and uses :code:`chirptime` to compute
+    the track.
+    """
     # Using findchirp_chirptime in PyCBC to calculate
     # the time-frequency track of dominant mode to get
     # the corresponding `f_min` for `t_obs_start`.
     freq_array = np.logspace(-4, 0, num=10)
     t_array = np.zeros(len(freq_array))
     for i in range(len(freq_array)):
-        t_array[i] = chirptime(m1=m1, m2=m2, f_lower=freq_array[i])
+        t_array[i] = chirptime(m1=m1, m2=m2, f_lower=freq_array[i], m_mode=m_mode)
```

Comment:
@mj-will Let's do a time-frequency plot check for all the modes. For example, use this function to calculate all the tf tracks for a SMBHB and a SOBHB, and plot them on two plots (one plot for SMBHB or SOBHB), to see if plots make sense.